test/pidfile.js

```diff
@@ -29,7 +29,7 @@ function noSuchPid() {
     try {
       process.kill(pid, 0);
     } catch(er) {
-      if(er.code == 'ESRCH') {
+      if(er.code == 'ESRCH' || er.message == 'invalid pid') {
```

Comment:
I think the proper fix here is to change `pid` to 0x7fffffff a few lines up.  `pid === (pid | 0)` should be true.